main.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime
+"""
+open all 3 files
+parse such that we have the log in a list
+count how many lines are in log, level=error count, level=info
+
+TIME FORMAT : 08:56:35.354
+
+top 5 most requested HTTP paths/endpoints
+"""
+
+class Logging:
+    def __init__(self):
+        self.log = []
+        self.date_time = []
+
+
+    def logs(self, file1, file2, file3):
```

Comment:
Doesn't matter too much in an interview context(It's more important to finish), but this function is named poorly. Functions usually should carry a verb. If they are missing a verb then I expect that verb to essentially be `get`. So when calling this function I actually expect it to give me back logs, which would be false here.

Preferable names here would be something like: `parse_logs` or `ingest_logs`